### ISSUE TITLE
Socialize IA proposals and integrate further feedback

### ISSUE THREAD

**Issue by cassiemc** [MozillaFoundation/plan#751]
#### RASCI
R: @cassiemc
A: @kristinashu 
S:
C: @secretrobotron @LauraReynal 
I:

#### Links
- [ ] Consider education hub #741
- [ ] Resolve open questions on IA doc: https://docs.google.com/presentation/d/1MOhf0PSvCU2flniNsbl1FW0xPi0ZnJHJJEZ3qlyAcrE/edit
- [ ] Review with @secretrobotron and @LauraReynal 
- [ ] Begin socializing with others
- [ ] probably something else

**Comment by secretrobotron** [comment]
:+1:

**Comment by secretrobotron** [comment]
Happy to help socialize so I can internalize this as well.

**Comment by cassiemc** [comment]
@simonwex Curious how you see this IA work integrating with the CMS? Should we set up domains/wrappers for each property that is likely to use the CMS (is that an eventual goal)?

**Comment by cassiemc** [comment]
@secretrobotron Just scheduled a catch up on this stuff for Monday, then hoping you can help me :)

**Comment by cassiemc** [comment]
Spoke to @secretrobotron and there are certain themes / assumptions we'd like to test with 10 people that involve the overall IA approach. We are writing up thoughts and a script here: https://docs.google.com/document/d/1qhvY6kn-1pGq0EqlHCQ9fWjhoxfVDs3tDpfB1QFJdys/edit

**Comment by cassiemc** [comment]
Ok! Final plan after talking with @secretrobotron and @samanthaburton in Toronto – we managed between me and @hannahkane to come up with some really solid questions/goals that we can use in the already-planned value prop research around the 'leadership network' idea that is being coordinated with Bobby, @LauraReynal and @adamlofting. That work will continue as planned. 

It doesn't seem necessary to duplicate our efforts on that in the sole context of the IA, though. If we phase the IA, we can essentially carry on as planned without blockers, and we can start filing tix to make non-controversial changes. (I've filed that as a followup issue here: https://github.com/MozillaFoundation/plan/issues/764)

There are [a few diagram variations as slides 2 & 3 in the IA doc](https://docs.google.com/presentation/d/1MOhf0PSvCU2flniNsbl1FW0xPi0ZnJHJJEZ3qlyAcrE/edit#slide=id.g11d935e442_0_129) that I hope to present to exec, along with some more context in the doc, to get feedback on next Wednesday.

Calling this one closed.
